src/inmanta/server/services/resourceservice.py

```diff
@@ -1231,6 +1231,19 @@ async def resource_list(
 
         # TODO: optimize for no orphans
 
+    @handle(methods_v2.resources_status, env="tid")
+    async def resources_status(
+        self,
+        env: data.Environment,
+        model_version: int,
+        rids: list[ResourceIdStr],
+    ) -> dict[str, ResourceState]:
```

Comment:
```suggestion
    ) -> dict[ResourceIdStr, ResourceState]:
```

Reply:
that produces a type error on the @handle annotation, so I had to choose